modules/reindex/src/test/java/org/opensearch/index/reindex/ReindexFromRemoteWhitelistTests.java

```diff
@@ -104,32 +104,32 @@ public void testWhitelistedByPrefix() {
                 RemoteInfo.DEFAULT_CONNECT_TIMEOUT
             )
         );
-        checkRemoteWhitelist(
-            buildRemoteWhitelist(singletonList("*.example.com:9200")),
+        checkRemoteAllowlist(
+            buildRemoteAllowlist(singletonList("*.example.com:9200")),
             newRemoteInfo("6e134134a1.us-east-1.aws.example.com", 9200)
         );
     }
 
     public void testWhitelistedBySuffix() {
-        checkRemoteWhitelist(buildRemoteWhitelist(singletonList("es.example.com:*")), newRemoteInfo("es.example.com", 9200));
+        checkRemoteAllowlist(buildRemoteAllowlist(singletonList("es.example.com:*")), newRemoteInfo("es.example.com", 9200));
     }
 
     public void testWhitelistedByInfix() {
-        checkRemoteWhitelist(buildRemoteWhitelist(singletonList("es*.example.com:9200")), newRemoteInfo("es1.example.com", 9200));
+        checkRemoteAllowlist(buildRemoteAllowlist(singletonList("es*.example.com:9200")), newRemoteInfo("es1.example.com", 9200));
     }
 
     public void testLoopbackInWhitelistRemote() throws UnknownHostException {
-        List<String> whitelist = randomWhitelist();
-        whitelist.add("127.0.0.1:*");
-        checkRemoteWhitelist(buildRemoteWhitelist(whitelist), newRemoteInfo("127.0.0.1", 9200));
+        List<String> allowlist = randomAllowlist();
+        allowlist.add("127.0.0.1:*");
+        checkRemoteAllowlist(buildRemoteAllowlist(allowlist), newRemoteInfo("127.0.0.1", 9200));
     }
 
     public void testUnwhitelistedRemote() {
         int port = between(1, Integer.MAX_VALUE);
-        List<String> whitelist = randomBoolean() ? randomWhitelist() : emptyList();
+        List<String> allowlist = randomBoolean() ? randomAllowlist() : emptyList();
         Exception e = expectThrows(
             IllegalArgumentException.class,
-            () -> checkRemoteWhitelist(buildRemoteWhitelist(whitelist), newRemoteInfo("not in list", port))
+            () -> checkRemoteAllowlist(buildRemoteAllowlist(allowlist), newRemoteInfo("not in list", port))
         );
         assertEquals("[not in list:" + port + "] not whitelisted in reindex.remote.whitelist", e.getMessage());
```

Comment:
Thank @nknize pointing out the reason to the test failure, seems the `whitelisted` is missed.